index.php

```diff
@@ -20,70 +20,79 @@
 
 /**
  * Retrieves a congiguration data, and converts it to an array.
- * 
- * @param  string $strPath
+ *
+ * @param string $strPath
  * @return array
  */
 function getConfig($strPath)
 {
     $arrRetVal = [];
-    
+
```

Comment:
In general - good idea!
I think it would be even better to add a default value to the input, like this:
<code>function getConfig($strPath = __DIR__ . '/config.json') {</code>